releases/release-1.13/release_team.md

```diff
@@ -0,0 +1,13 @@
+| **Role** | **Name** (**GitHub / Slack ID**)  | **Shadow Name(s) (GitHub / Slack ID)** |
+| ------ | ------ | ------ |
+| Lead | Aishwarya Sundar ([@AishSundar](https://github.com/AishSundar)) | |
```

Comment:
Please add Aaron Crickenberger (@spiffxp) as the Release lead shadow

Reply:
Thanks!